src/OVAL/probes/SEAP/public/sm_alloc.h

```diff
@@ -35,36 +36,36 @@ extern "C" {
 
 #if !defined(HAVE_POSIX_MEMALIGN)
 # if defined (HAVE_MEMALIGN)
-extern int posix_memalign (void ** __memptr, size_t __alignment, size_t __size);
+OSCAP_API extern int posix_memalign (void ** __memptr, size_t __alignment, size_t __size);
```

Comment:
woah, are you sure about this?

Reply:
actually I myself am not sure but I find it suspicious that something this low level is exported, it's also something that is defined in other SOs with the same symbol name so this might cause clashes, and it's defined as extern. I am basically asking if you had good reasons to explicitly export it.

Reply:
That's because this is a header file in a public directory. I haven't noticed it's in rbt_common.h. Are there some public headers that aren't public?

Reply:
I don't know. I think we will need to review symbols in the public API before we release 1.3.0, because there can be more functions in public files that should be private.

For now I'll add a patch that removes OSCAP_API from this line. I don't want to do bigger changes of header files in this PR, because it's already a big PR.